store/sqlstore/channel_store.go

```diff
@@ -1558,6 +1558,21 @@ func (s SqlChannelStore) GetDeleted(teamId string, offset int, limit int, userId
 	return channels, nil
 }
 
+var channelMembersForTeamWithSchemeSelectQueryX = sq.Select(
+	"ChannelMembers.*",
+	"TeamScheme.DefaultChannelGuestRole TeamSchemeDefaultGuestRole",
+	"TeamScheme.DefaultChannelUserRole TeamSchemeDefaultUserRole",
+	"TeamScheme.DefaultChannelAdminRole TeamSchemeDefaultAdminRole",
+	"ChannelScheme.DefaultChannelGuestRole ChannelSchemeDefaultGuestRole",
+	"ChannelScheme.DefaultChannelUserRole ChannelSchemeDefaultUserRole",
+	"ChannelScheme.DefaultChannelAdminRole ChannelSchemeDefaultAdminRole",
+).
+	From("ChannelMembers").
+	InnerJoin("Channels ON ChannelMembers.ChannelId = Channels.Id").
+	LeftJoin("Schemes ChannelScheme ON Channels.SchemeId = ChannelScheme.Id").
+	LeftJoin("Teams ON Channels.TeamId = Teams.Id").
+	LeftJoin("Schemes TeamScheme ON Teams.SchemeId = TeamScheme.Id")
+
 var channelMembersForTeamWithSchemeSelectQuery = `
```

Comment:
I know this is out of scope, but I am wondering if we can take this opportunity to completely replace this string with the sqlx query everywhere?

Reply:
Sure, I'll get right to it.

Reply:
Done, I've migrated all methods using this query to Squirrel.